docs/release-notes.rst

```diff
@@ -1,3 +1,216 @@
+Release Notes (Version 2.6)
+===========================
+
+General
+-------
+
+**File system convergence**
+
+APIC Filesystem Glusterfs takes 15 to 20 Minutes to recover when APIC cluster goes unhealthy
+
+During APIC operations like APIC reboot, upgrade, or decommission/recommission, the APIC filesystem needs some time to recover and resume.
+
+Symptoms users may see:
+1. Application operations throw a (sqlite3.OperationalError) disk I/O error
+2. Users may not find previously added BIG-IP entries in the application
+3. New Application installation may fail
+
+**Workaround**: Wait for 20 minutes and try to access the application again. The Glusterfs should recover automatically and the application should be accessible again.
+
+------
+
+**Floating IP auto sync and Default Gateway auto sync will not work when hosts are added in an HA cluster using hostnames**
+
+**Workaround:** Manually sync the Floating IPs and Default Gateway to Application DB by clicking on **Sync To DB**
+
+------
+
+**Export db does not work on APIC version 5.0.1**
+
+The user menu on the top-right corner of the F5 ACI ServiceCenter has an option **Export/Import DB > Export DB**, for exporting the F5 ACI ServiceCenter database. This option does not work on APIC versions 5.0.1. This is working as expected on prior versions of the APIC and also on versions 5.0(2e) and higher. 
+
+**Workaround:** Backup the F5 ACI ServiceCenter database via the following steps:
+1. Login to the APIC server via SSH
+
+2. cd /data2/gluster/gv0/F5Networks_F5ACIServiceCenter
+
+3. Use scp, winscp or any other preferred tool to copy out the f5.db file from this location. 
+
+------
+
+**Non-default docker0 configuration on APIC may affect connection to the management IP of Virtual BIG-IP**
+
+The default docker0 bridge IP has the address **172.17.0.1**. If the docker0 bridge IP is updated to a non-default IP followed by APIC cluster reboot, the Virtual BIG-IP login (connection to management IP) will fail from the F5 ACI ServiceCenter.
+
+**Workaround:** After APIC reboots, once again change the docker0 bridge IP to a new value.
+
+------
+
+**Zoom In/Out for Device Discovery Topology diagrams may get stuck on Internet Explorer**
+
+Zoom In and Zoom out of Topology diagrams which open on Topology icon click in every tab (Visibility, Network Management and L4-L7 App Services) may get stuck in Internet Explorer browser. The position of the topology diagram can also not be changed once this issue is observed. 
```

Comment:
this first sentence is a little confusing, maybe something like:

"When you click the Topology icon in all tabs (Visibility, Network Management, and L4-L7 App Services), the Topology diagrams may get stuck when zooming in and out."